internal/pkg/agent/install/install.go

```diff
@@ -216,6 +207,27 @@ func Install(cfgFile, topPath string, unprivileged bool, log *logp.Logger, pt *p
 	return ownership, nil
 }
 
+// setup the basic topPath, and the .installed file
+func setupInstallPath(topPath string, ownership utils.FileOwner) error {
+	// ensure parent directory exists
+	err := os.MkdirAll(filepath.Dir(topPath), 0755)
+	if err != nil {
+		return errors.New(err, fmt.Sprintf("failed to create installation parent directory (%s)", filepath.Dir(topPath)), errors.M("directory", filepath.Dir(topPath)))
+	}
+
+	// create Agent/ directory with more locked-down permissions
+	err = os.MkdirAll(topPath, 0750)
```

Comment:
Why do we need to do this now? It might have been discussed already but I missed it.

Reply:
So, this is normally performed by the later `Copy` operation that does the actual file copying. However, we're now creating the directory before we call `Copy`, so we need to create the directory manually.